modules/apps/translation/translation-translator-aws/src/main/java/com/liferay/translation/translator/aws/internal/translator/AWSTranslator.java

```diff
@@ -50,13 +49,36 @@ public class AWSTranslator implements Translator {
 
 	@Override
 	public boolean isEnabled(long companyId) throws ConfigurationException {
-		return _awsTranslatorConfiguration.enabled();
+		AWSTranslatorConfiguration awsTranslatorConfiguration =
+			_configurationProvider.getCompanyConfiguration(
+				AWSTranslatorConfiguration.class, companyId);
+
+		return awsTranslatorConfiguration.enabled();
 	}
 
 	@Override
 	public TranslatorPacket translate(TranslatorPacket translatorPacket)
 		throws PortalException {
 
+		AWSTranslatorConfiguration awsTranslatorConfiguration =
+			_configurationProvider.getCompanyConfiguration(
+				AWSTranslatorConfiguration.class,
+				translatorPacket.getCompanyId());
+
+		if (awsTranslatorConfiguration.enabled()) {
```

Comment:
Isn't this condition inverted? As it is, this will do nothing when AWS translator is enabled.